modules/uhsg_rest/src/Plugin/rest/resource/DegreeProgrammeResource.php

```diff
@@ -147,6 +157,8 @@ private function getNodeCount(TermInterface $term, $content_type) {
         ->condition('field_news_degree_programme', $term->tid->value)
         ->condition('type', $content_type)
         ->condition('langcode', $languageCode)
+        ->accessCheck(false)
+        ->status(true)
```

Comment:
This is throwing an error: `Error: Call to undefined method Drupal\Core\Entity\Query\Sql\Query::status()`. Maybe it should be something like `->condition('status', 1)`?

Reply:
Sorry, Im stupid. Apparetly didnt clear opcache or something when I tested this. Should be fixed now.